File: server-dotNet/Com.EnuoCms.Web/Controllers/AdminController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using Com.EnuoCms.IService;
using Com.EnuoCms.Model.Entities;

/**
 * Created by zhanxiaoping 
 * zhanxp@me.com
 */
namespace Com.EnuoCms.Web.Controllers
{
	[RoutePrefix("admin")]
    public class AdminController : Controller
    {
		private readonly IAdminRepository adminRepository;

		public AdminController(
			IAdminRepository adminRepository
			)
		{
			this.adminRepository = adminRepository;
		}

        [Route]
        public ActionResult Index()
        {
            var userName = User.Identity.Name;
            Admin user = adminRepository.Get(sim => sim.Name == userName);
            return View("Profile", user);
        }
    }
}

File: server-spring-mvc/enuocms-web/src/main/java/com/enuocms/web/controller/AdminController.java
package com.enuocms.web.controller;


import com.enuocms.account.admin.Admin;
import com.enuocms.account.admin.AdminService;
import com.enuocms.web.shiro.ShiroUser;
import com.enuocms.web.util.UserUtil;
import lombok.extern.slf4j.Slf4j;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * Created by zhanxiaoping on 2017/8/28.
 * zhanxp@me.com
 */
@Controller
@RequestMapping("admin")
@Slf4j
public class AdminController {

    @Autowired
    AdminService adminService;

    @RequestMapping("")
    public String index(Model model) {
        ShiroUser user = UserUtil.getCurrentUser();

        Admin profile = adminService.get(user.getId());
        model.addAttribute("profile",profile);

        return "/admin/profile";
    }
}
